sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -432,6 +432,17 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 		log.G(ctx).Info("successfully started container task")
 	}
 
+	// Track the task's PID namespace in a manifest so monitoring agents can access it
+	taskPidNs, pidNsErr := getTaskPidNs(&task)
+	if pidNsErr != nil {
```

Comment:
I'm fine with this, but looks like typically just call these `err` in the previous code although this is more clear.

Reply:
Reason why I keep it separate here is because I need to check later to make sure the previous `getTaskPidNs` call succeeded before I remove the PID NS from the manifest file after the task has exited:
https://github.com/bottlerocket-os/bottlerocket/blob/2401a7679181f9479f97252fc79957a549770ad4/sources/host-ctr/cmd/host-ctr/main.go#L504-L510

Reply:
Ahh, I missed that part 👏🏼 good thinking